neuralcoref/train/learn.py

```diff
@@ -303,8 +303,8 @@ def run_epochs(start_epoch, end_epoch, loss_func, optim_func, save_name, lr, g_s
     parser.add_argument('--min_lr', type=float, default=2e-8, help='min learning rate')
     parser.add_argument('--on_eval_decrease', type=str, default='nothing',
                         help='What to do when evaluation decreases ("nothing", "divide_lr", "next_stage", "divide_then_next")')
+    parser.add_argument('--lazy', action='store_true', help='Use lazy loading while loading the npy files')
```

Comment:
I would define the type, default and make it clear in the description:
`type=int, default=1, help='Use lazy loading (1, default) or not (0) while loading the npy files'`

I think this setting can be on by default :-)

Reply:
Done!